pkg/generator_test.go

```diff
@@ -740,3 +741,11 @@ func TestParseReplaceType(t *testing.T) {
 		assert.Equal(t, test.expected, *actual)
 	}
 }
+
+func (s *GeneratorSuite) TestGeneratorVariadicTester() {
+	expectedBytes, err := os.ReadFile(getMocksPath("VariadicTester.go"))
+	s.Require().NoError(err)
+	expected := string(expectedBytes)
+	expected = expected[strings.Index(expected, "// VariadicTester is"):]
+	s.checkGeneration("variadic_tester.go", "VariadicTester", false, "", expected)
```

Comment:
We should not do string comparison tests because it's brittle and tests the wrong thing anyway. Just make sure that the produced mock object itself can be used and that it satisfies the interface that we expect for `VariadicTester`.

That may make your test coverage "technically" go down but codecov isn't smart enough to realize we are in fact getting the proper testing coverage here.